File: src/model/FinancialAsset.ts
import { FinancialType } from "./enums/FinancialType";
 export class FinancialAsset{
        ticker:string
         amount:number;
         type:FinancialType;
         name:string;
         quanity:number;


    constructor(amount:number,type:FinancialType,name:string,quantity:number,ticker:string){
            this.amount = amount;
            this.type = type;
            this.name = name;
            this.quanity = quantity;
            this.ticker = ticker;
    }

    
}

File: money-bot-refactor/src/main/java/com/money_bank/gui/neg/git/project/model/FinancialAsset.java
package com.money_bank.gui.neg.git.project.model;

import com.money_bank.gui.neg.git.project.model.enums.FinancialType;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class FinancialAsset {
    private String ticker;
    private BigDecimal amount;
    private FinancialType financialType;
    private String name;
    private double quantity;

}
